### PR TITLE
Improve snippets test coverage for tasks

### PR DESCRIPTION
**Pull request by 6hundreds**
Part of https://github.com/gradle/gradle/issues/21027

### CODE REVIEW

File: subprojects/docs/src/snippets/tasks/inputNormalizationMetaInf/tests/runtimeClasspathNormalization.sample.conf [hunk 1]
```diff
@@ -0,0 +1,2 @@
+executable: gradle
+args: tasks
```
Comment (mlopatkin) [inline subprojects/docs/src/snippets/tasks/inputNormalizationMetaInf/tests/runtimeClasspathNormalization.sample.conf:2]: The snippet has the `java` plugin applied, maybe run `check` here instead?

**Review by mlopatkin — APPROVED** [review]
Thanks! LGTM with a nit.

File: subprojects/docs/src/snippets/tasks/inputNormalizationMetaInf/tests/runtimeClasspathNormalization.sample.conf (hunk 1 above)
Comment (6hundreds) [reply subprojects/docs/src/snippets/tasks/inputNormalizationMetaInf/tests/runtimeClasspathNormalization.sample.conf]: Agree, done

**Comment by 6hundreds** [comment]
@bot-gradle test and merge

**Review by bamboo — COMMENTED** [review]
👍

**Pull request merged**